src/components/Badge/Badge.module.css

```diff
@@ -22,7 +22,7 @@
    */
   --badge-height: calc(var(--eds-size-2-and-half) + 2 * var(--eds-border-width-sm));
 
-  z-index: var(--eds-z-index-100);
+  z-index: 100;
```

Comment:
Generally KISS'd away any token usages where the token had the value in the token name. What good is a token where the name is NOT the value?